parm/evs_config/analyses/config.evs.rtma.prod

```diff
@@ -44,9 +44,9 @@ export end_date=$yday
 #metplus_verbosity:         DEBUG, INFO, WARN, ERROR (most verbose is DEBUG)
 #met_verbosity:             MET verbosity for logging, 1-5 (most verbose is 5)
 #log_met_output_to_metplus: log MET output to METplus log, yes or no
-export metplus_verbosity="DEBUG"
-export met_verbosity="2"
-export log_met_output_to_metplus="yes"
+#export metplus_verbosity="DEBUG"
+#export met_verbosity="2"
+#export log_met_output_to_metplus="yes"
 ## DATA DIRECTIVE SETTINGS
```

Comment:
Do you want lines 47-49 formally removed, or keep them in as commented out? Same question for the equivalent parm file for urma.

Reply:
Removed.  Let me fix that.

Reply:
Fixed.